docs/src/main/sphinx/connector/non-transactional-merge.fragment

```diff
@@ -0,0 +1,10 @@
+### Non-transactional MERGE
+
+The connector supports adding rows using [MERGE statements](/sql/merge), if the
```

Comment:
`The connector supports adding rows using` MERGE can add/update/delete rows, maybe you have better phrase? like change `adding rows` to `modifying rows`?

Reply:
Updated